xcp_d/workflows/bold/metrics.py

```diff
@@ -145,7 +145,6 @@ def init_alff_wf(
         ComputeALFF(TR=TR, low_pass=low_pass, high_pass=high_pass),
         mem_gb=mem_gb["resampled"],
```

Comment:
I hope the memory limits will be enough to prevent nipype from running too many of these at once